src/models/gorilla.cpp

```diff
@@ -2,6 +2,7 @@
 #include <math.h>
 #include <iostream>
 #include "../doctest.h"
+#include <math.h>
```

Comment:
`math.h` er deprecated i C++, brug `cmath` i stedet for.

Reply:
Er heller ikke i brug.

Reply:
Jo, er i brug, den er bare included to gange. Fixed